src/lib/tailwind/tokens/rings.cjs

```diff
@@ -0,0 +1,42 @@
+// Design Tokens: Rings
+// Doc: https://www.skeleton.dev/elements/tokens
+
+const settings = require('../settings.cjs');
+
+// Local
+const ringOutlineShared = {
+    // .ring-[1px]
+    '--tw-ring-offset-shadow': 'var(--tw-ring-inset) 0 0 0 var(--tw-ring-offset-width) var(--tw-ring-offset-color)',
+    '--tw-ring-shadow': 'var(--tw-ring-inset) 0 0 0 calc(1px + var(--tw-ring-offset-width)) var(--tw-ring-color)',
+    'box-shadow': 'var(--tw-ring-offset-shadow), var(--tw-ring-shadow), var(--tw-shadow, 0 0 #0000)',
+    // .ring-inset
+    '--tw-ring-inset': 'inset'
+};
+
+module.exports = () => {
+    const classes = {
+        // Ring Outline (for cards)
+        // Example: .ring-outline-token
+        '.ring-outline-token': {
+            ...ringOutlineShared,
+            '--tw-ring-color': 'rgb(23 23 23 / 0.05);', // neutral-900
```

Comment:
can we extract the value to a `const` and name it so we know what 23 23 23 / 0.05 is?